lib/checks/aria/required-children.js

```diff
@@ -94,6 +95,15 @@ function missingRequiredChildren(node, childRoles, all, role) {
 	return null;
 }
 
+function hasChildWithRole(node) {
+	return (
+		node.children &&
+		node.children.some(child => {
+			return axe.commons.aria.getRole(child) || hasChildWithRole(child);
```

Comment:
This should ignore role=none/presentation. Should we add an option to `getRole` so that it returns `null` if the role is `presentation` or `none`? We seem to be doing a lot of this: `!['presentation', 'none'].includes(role)`

Maybe we can make this:

```js
getRole(elm, { noPresentation: true })
```